server/command.go

```diff
@@ -122,12 +123,27 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 			p.postCommandResponse(args, txt)
 			return &model.CommandResponse{}, nil
 		} else if len(parameters) > 1 {
-			features = strings.Join(parameters[1:], " ")
+			featureList := []string{}
+
+			for _, element := range parameters[1:] {
+				if isFlag(element) {
+					flags = append(flags, parseFlag(element))
+				} else {
+					featureList = append(featureList, element)
+				}
+			}
+
+			if len(featureList) > 1 {
+				p.postCommandResponse(args, "Just one {features} parameter is allowed")
```

Comment:
Hello @marianunez, thanks for your comments!

I don't actually know if what the expected behaviour is. Currently we're allowing:

`/github subscribe {owner} {features}`

What we're checking looping over the expression is basically that one (or none) parameter is supplied (in this case "features", which is a comma-separated string). I mean, this command:

`/github subscribe repo issues,pulls anotherThing --exclude-org-member`

shouldn't be valid. With the previous implementation we would be inserting in the `Subscription` struct:

`Features: issues,pulls anotherThing`

but "Features" is just the command-separated string.

I don't know if this makes sense

Thanks!

Reply:
I see. Agreed that before it was not taking that into account and was taking everything after the first parameter. Makes sense your implementation then 👍

Reply:
Having a look at it again, I think the naming might be confusing. Instead of featureList we should use parameterList.

I'll change that too if you're ok with it

Reply:
Yes I guess `featureList` now is not enough considering we are adding flag support too. However, parameterList not sure if it will be confused with the whole list of initial `parameters` [here](https://github.com/mattermost/mattermost-plugin-github/blob/b89d7b68e4c5b8760c76df45bcc207f4521131a7/server/command.go#L59)

Reply:
Maybe optionList?

Once you are inside a command (in this case, "subscribe"), you have "options" (in this case, only features)

What do you think?

Reply:
`optionList` sounds better 💯

Reply:
Great! Done. Thanks a lot.